modules/guardduty-org-admin-account/main.tf

```diff
@@ -0,0 +1,36 @@
+# Creates a GuardDuty org administrator account in an AWS organization.  Once created, all AWS accounts added to the AWS organization will be enrolled in GuardDuty and managed by the delegated GuardDuty org administrator account
+# - Creates a GuardDuty detector for the org's GuardDuty administrator account
+# - Creates a GuardDuty organization administrator account
+# - Configures the delegated GuardDuty administrator organization configuration
+#
+# Prerequisites:  The AWS org must already exist
+
+# Create GuardDuty detector for the organization's GuardDuty administrator account
+resource "aws_guardduty_detector" "guardduty_administrator" {
```

Comment:
I would suggest removing this resource, and taking the attributes you are using later as variables. The user can use the top-level module to fully configure the guardduty detector in the administrator account.

Reply:
I like your perspective.  Change made.